docs/user/security/authentication/index.asciidoc

```diff
@@ -144,9 +150,11 @@ Note that with `server.ssl.clientAuthentication` set to `required`, users are as
 [[saml]]
 ==== SAML single sign-on
 
-SAML authentication allows users to log in to {kib} with an external Identity Provider, such as Okta or Auth0. Make sure that SAML is enabled and configured in {es} before setting it up in {kib}. See {ref}/saml-guide.html[Configuring SAML single sign-on on the Elastic Stack].
+SAML authentication is part of single sign-on (SSO), a https://www.elastic.co/subscriptions[subscription feature]. This allows users to log
+in to {kib} with an external Identity Provider, such as Okta or Auth0. Make sure that SAML is enabled and configured in {es} before setting
+it up in {kib}. See {ref}/saml-guide.html[Configuring SAML single sign-on on the Elastic Stack].
```

Comment:
General note for this and other SSO sections (OIDC, Kerberos): I changed them each to state "single sign-on (SSO)" at the beginning and then use SSO for the rest of the section.